pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -54,19 +55,31 @@ var _ webhook.Validator = &Cluster{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *Cluster) ValidateCreate() error {
 	clusterlog.Info("validate create", "name", r.Name)
-	if r.IsReconcilePaused() {
-		clusterlog.Info("cluster is paused, so allowing create", "name", r.Name)
-		return nil
-	}
+	var allErrs []error
+
 	if !features.IsActive(features.FullLifecycleAPI()) {
```

Comment:
I don't think this is correct. This will add the error even for self-managed clusters